### DISCUSSION TITLE
collector error

### DISCUSSION THREAD

**Discussion by quantum-decrypt-security** [cloudforet-io/community#130] [Q&A]
The following error occurs when creating a collector. I would like to know what is causing this issue.

`Something Is Wrong! Please Contact The Administrator.
Fail to verify plugin, params={'name': 'bvbnvbn', 'plugin_info': {'plugin_id': 'plugin-49f224ef6d36', 'provider': 'aws', 'upgrade_mode': 'AUTO'}, 'priority': 10, 'tags': {}, 'domain_id': 'domain-fd424f1e5de8', 'capability': {'supported_schema': ['aws_access_key', 'aws_assume_role', 'aws_assume_role_with_external_id']}, 'provider': 'aws'}`

**Comment by jihyungSong** [comment]
Hello, guys! 

This error message appears to have occurred during the installation of a collector plugin, and it seems to have failed during the verification step after being deployed a container internally. Would you inform me about the type of collector that was being deployed?

It would be good to verify if the container corresponding to that collector was actually deployed in Kubernetes. Checking the pods deployed in the "spaceone-plugin" namespace would be a good step. Additionally, check the logs within the pod can help determine why the verification failed.

**Reply by quantum-decrypt-security** [reply]
Hello @jihyungSong !
When I checked spaceone-plugin, nothing was related to the cloud(AWS,GCP,AZURE). I need to install additional plugins; please guide me.

NAME                                                              READY   STATUS    RESTARTS        AGE
plugin-ms-teams-noti-protocol-aobbcnfmuvvcolcl-5cb6c867db-w4p5m   1/1     Running   6 (12h ago)     12h
plugin-ms-teams-noti-protocol-dovvcefrsghxrsn-5b4b95684d-jgtk8    1/1     Running   9 (6h4m ago)    6h25m
plugin-ms-teams-noti-protocol-orzziefluzizzcx-78664548fb-jwp5t    1/1     Running   0               22h
plugin-ms-teams-noti-protocol-znooimfmskhdlhbm-5487f46b8b-vhkfz   1/1     Running   14 (6h2m ago)   6h45m
plugin-telegram-noti-protocol-rrzzizfzuviozuqv-5d469ccd46-6zvsz   1/1     Running   0               22h
slack-notification-protocol-kgootkfguvivvcob-557658b459-d59s8     1/1     Running   0               22h
